terranodo/eventkit-cloud
# Make banner configurable

This pr makes the banner configurable. It converts the disclaimer endpoint into a generic config endpoint which serves a json of whatever ui configs are added to the settings file. The application component will get the json on mount and add the config to child contexts. 
To test:
Rebuild httpd
Add values to the disclaimer and banner env vars in the docker-compose file
Ensure those changes are reflected in the ui.
![image](https://user-images.githubusercontent.com/16799449/27041344-bc9cee0a-4f61-11e7-921e-9f3e636af135.png)
